cg/src/domains/processors/processors.listeners.js

```diff
@@ -2,15 +2,15 @@ const { listenForErasureRequest, listenForConsent } = require('./../../utils/blo
 const { getDataForSubject } = require('./processors.requests');
 const { blockUntilContractReady } = require('./processors.helpers');
 const { inControllerMode } = require('../../utils/helpers');
-const SubjectService = require('./../subjects/subjects.service');
+const SubjectSService = require('./../subjects/subjects.service');
```

Comment:
And what's happened with the casing here?
Was anything wrong with `SubjectService`?

Reply:
was missing an 's', but I put an uppercase one, fixing it